src/components/book.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Button from './button';
+
+const Book = () => (
```

Comment:
- [ ] The **Book** component should take in a book prop data and render the book title and author